main/swadge2024.c

```diff
@@ -229,8 +230,8 @@ void app_main(void)
     // Init buttons and touch pads
     gpio_num_t pushButtons[] = {
         GPIO_NUM_0,  // Up
-        GPIO_NUM_4,  // Down
-        GPIO_NUM_2,  // Left
+        GPIO_NUM_2,  // Down
+        GPIO_NUM_4,  // Left
```

Comment:
Note to self, undo this eventually

Reply:
@MrTroyD I undid this in your branch before merging. Feel free to redo it for your board, or leave it as an uncommitted change